client/components/header-nav/README.md

```diff
@@ -0,0 +1,4 @@
+Header Nav
```

Comment:
Alright, I'm going with ```SubMasterbarNav```.  
Using just ```SubMasterbar```might prove problematic in the future in case we'd want to put  something else up there.